src/NServiceBus.AcceptanceTests/Core/Outbox/When_outbox_double_optin_found.cs

```diff
@@ -7,28 +7,26 @@ namespace NServiceBus.AcceptanceTests.Core.Outbox
     using EndpointTemplates;
     using Logging;
     using NUnit.Framework;
-    using ScenarioDescriptors;
 
     public class When_outbox_double_optin_found : NServiceBusAcceptanceTest
     {
         [Test]
-        public Task Should_log_warning()
+        public async Task Should_log_warning()
         {
+            Requires.OutboxPersistence();
```

Comment:
we don't check here whether outbox is enabled or not, we just ensure that an outbox capable storage is available, that's why I went with the current proposal. No strong opinion though.